DevCycle.SDK.Server.Common/API/DVCBaseClient.cs

```diff
@@ -52,13 +52,13 @@ protected void ValidateUser(User user)
             }
         }
 
-        protected async Task<T> GetResponseAsync<T>(object body, string urlFragment, Dictionary<string, string> queryParams = null)
+        protected async Task<T> GetResponseAsync<T>(object body, string urlFragment, Dictionary<string, string> queryParams = null, bool shouldRetry = true)
```

Comment:
it seems a bit inconsistent across these methods whether this should default to true or false. Is there a reason for it to be different here than in the API base client?

Reply:
The `GetResponseAsync` is used only by the cloud sdk so I thought we'd want the default to be always retry. Whereas the `SendRequestAsync` is used by the DVCEventsAPIClient for the local sdk so I thought I'd default it to false there. I can change the `SendRequestAsync` to also be true and just pass in false if that would make it more clear

Reply:
I think its fine I was just wondering